<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>RegularExpressionValidator</class>
  <widget class="QMainWindow" name="RegularExpressionValidator">
    <property name="minimumSize">
      <size>
        <width>800</width>
        <height>600</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>RegularExpressionValidator</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
          <widget class="QComboBox" name="comboBox_MatchRules" />
        </item>
        <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_RegularExpression">
            <property name="placeholderText">
              <string>请在此输入正则表达式</string>
            </property>
          </widget>
        </item>
        <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_Validate">
            <property name="text">
              <string>验证</string>
            </property>
          </widget>
        </item>
        <item row="1" column="0" colspan="3">
          <widget class="QTextEdit" name="textEdit_MatchContent" />
        </item>
      </layout>
    </widget>
    <widget class="QStatusBar" name="statusbar" />
  </widget>
  <resources />
  <connections />
</ui>
